worldedit-core/src/main/java/com/sk89q/worldedit/registry/Registry.java

```diff
@@ -46,6 +54,10 @@ public String getName() {
     @Nullable
     public V get(final String key) {
         checkState(key.equals(key.toLowerCase(Locale.ROOT)), "key must be lowercase: %s", key);
+        if (this.checkInitialized) {
+            checkState(WorldEdit.getInstance().getPlatformManager().isInitialized(),
+                    "WorldEdit is not enabled yet.");
```

Comment:
```suggestion
                    "WorldEdit is not initialized yet.");
```
I would prefer consistent wording here, especially because "enabled" is a Bukkit-specific term.